buildSrc/src/main/kotlin/io/provenance/DownloadProtosPlugin.kt

```diff
@@ -0,0 +1,21 @@
+package io.provenance
+
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.kotlin.dsl.register
+
+/**
+ * Custom gradle plugin to download Provenance and Cosmos protobuf files.
+ *
+ */
+class DownloadProtosPlugin: Plugin<Project> {
+    override fun apply(project: Project) {
+        project.tasks.register(
+            "downloadProtos",
+            DownloadProtosTask::class
+        ) {
+            this.group = "help"
+            this.description = "Downloads Provenance and Cosmos protobuf files.  Specify the Provenance and Cosmos versions: --provenance-version v0.2.1 --cosmos-version v0.42.2"
```

Comment:
Will this need to be updated every time we bump teh versions? Or can we tie this to the defined versions?

Reply:
Yes, it will.  There's no clean way to find the latest version and I'm of the opinion that the downloader should specify versions manually.